app/src/main/kotlin/com/hippo/ehviewer/ui/scene/FavoritesScene.kt

```diff
@@ -150,11 +151,10 @@ class FavoritesScene : SearchBarScene() {
     private val vm: VMStorage by viewModels()
     private var urlBuilder by lazyMut { vm::urlBuilder }
     private var _binding: SceneFavoritesBinding? = null
-    private val binding get() = _binding!!
     private var mAdapter: GalleryAdapter? = null
-    private val tracker get() = mAdapter!!.tracker!!
     private val showNormalFabsRunnable = Runnable {
-        updateJumpFab() // index: 0, 2
+        val binding = _binding ?: return@Runnable
```

Comment:
Did the NPE you encountered happen here?

Reply:
Yes,In this Runnable scope, binding should check null or not.

Reply:
If you do not want change binding，I can use coroutine to fix this code

Reply:
Yeah cancelling the job on lifecycle destroyed is the correct way.

Reply:
please check #152